src/collider/README.md

```diff
@@ -23,10 +26,12 @@ A websocket-based signaling server in Go.
 
 5. Install dependencies
 
-        go get collidermain
+        go get golang.org/x/net/websocket
```

Comment:
Are these really necessary since collidermain imports collider which has all the deps needed?

Reply:
yes it is because 'go get collidermain' fails for so many people despite having working go environments and configurations per instructions